test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs

```diff
@@ -728,6 +742,22 @@ private string PublishMSBuild(string projectDirectory, string projectName, strin
             return result.StdOut;
         }
 
+        private string PackMSBuild(string projectDirectory, string projectName)
```

Comment:
I'm not sure why we're introducing this pattern. new PackCommand().Execute.Should().Succeed() is much easier to understand than this custom helper.